docs/user/spark/jts_spark.rst

```diff
@@ -0,0 +1,103 @@
+JTS Spark
```

Comment:
i think you need to add an entry into docs/user/spark/index.rst for this to show up